src/Raven.Server/Config/Attributes/ConfigurationEntryAttribute.cs

```diff
@@ -12,11 +12,12 @@ namespace Raven.Server.Config.Attributes
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = true)]
     public class ConfigurationEntryAttribute : Attribute
     {
-        public ConfigurationEntryAttribute(string key, ConfigurationEntryScope scope, [CallerLineNumber]int order = 0, bool setDefaultValueIfNeeded = true) // the default order is the order of declaration in a configuration class
+        public ConfigurationEntryAttribute(string key, ConfigurationEntryScope scope, [CallerLineNumber]int order = 0, bool setDefaultValueIfNeeded = true, bool isSecured = false) // the default order is the order of declaration in a configuration class
         {
             Key = key;
             Order = order;
             SetDefaultValueIfNeeded = setDefaultValueIfNeeded;
+            IsSecured = isSecured;
```

Comment:
I don't understand the purpose of this? Are we talking about private information? Such as passwords?

Reply:
It is for the UI, so we can substitute the value with `***********` with 'button' to see the real value

Reply:
Why? The arguments, I can understand, but the exec doesn't seems like it should be.

Reply:
Depends on the angle. I prefer to hide it, because it reveals a path.

Reply:
Done